source/agora/consensus/data/ConsensusParams.d

```diff
@@ -50,19 +59,23 @@ public immutable class ConsensusParams
             validator_cycle = cycle length for a validator
             max_quorum_nodes = max nodes to include in autogenerated quorum set
             quorum_threshold = the threshold to use for the generated quorums
-            quorum_shuffle_interval = quorum shuffle block cycle length
+            genesis_start_time = unix timestamp of the Genesis block
 
     ***************************************************************************/
 
     public this (immutable(Block) genesis, uint validator_cycle = 1008,
                  uint max_quorum_nodes = 7, uint quorum_threshold = 80,
-                 uint quorum_shuffle_interval = 30)
+                 uint quorum_shuffle_interval = 30,
+                 time_t genesis_start_time = 1596179709,
```

Comment:
```suggestion
                 time_t genesis_start_time = 1596153600,
```

`assert(SysTime(DateTime.fromISOString("20200731T000000"), UTC()).toUnixTime == 1596153600);`

According to the time declared in config value, the unix timestamp value is as above.

Reply:
Ah yes. In any case these are just placeholder values. We need to change them to the actual time of when mainnet will be deployed. But we don't know this yet.